ambry-store/src/test/java/com.github.ambry.store/DiskSpaceAllocatorTest.java

```diff
@@ -0,0 +1,423 @@
+/*
```

Comment:
tests look great - I might be missing it but is there a test for `DiskSpaceAllocator` being used when pooling is disabled (construct, start, allocate, free)?

Reply:
I will add one in.